easybuild/easyconfigs/v/Visit/Visit-2.12.0-CrayGNU-2016.11.eb

```diff
@@ -0,0 +1,70 @@
+# contributed by Victor Holanda Rusu (CSCS)
+#            and Jean Favre (CSCS)
+#
+easyblock = 'CMakeMake'
+
+name = 'Visit'
+version = "2.12.0"
+
+homepage = 'https://wci.llnl.gov/codes/visit/'
+description = """VisIt is an Open Source, interactive, scalable, visualization,
+animation and analysis tool."""
+
+toolchain = {'name': 'CrayGNU', 'version': '2016.11'}
+toolchainopts = {'usempi': True, 'verbose': False}
+
+source_urls = ['http://portal.nersc.gov/project/visit/releases/%(version)s' ]
+sources = [SOURCELOWER_TAR_GZ]
+#sources = ['visit2.12.0.tar.gz']
+
+#buildininstalldir = False
+
+srcdir='../src'
+preconfigopts  = ' export PAR_COMPILER=`which cc`; '
+preconfigopts += ' export PAR_INCLUDE="-I$MPICH_DIR/include"; '
+preconfigopts += ' ./src/svn_bin/build_visit --console --no-visit '
+preconfigopts += ' --system-cmake --h5part --no-pyside --system-python --alt-boost-dir $EBROOTBOOST '
+preconfigopts += ' --vtk --mesa --parallel --hdf5 --silo --szip --makeflags -j20 '
+preconfigopts += ' --netcdf --qt <<< "yes" && '
+preconfigopts += ' sed -i /VISIT_MPI_COMPILER/d $HOSTNAME.cmake; '
+preconfigopts += ' cp $HOSTNAME.cmake ../visit%(version)s/src/config-site; '
+preconfigopts += ' mkdir build; '
+preconfigopts += ' cd build; '
+
+configopts  = ' -DCMAKE_BUILD_TYPE=RelWithDebInfo '
+configopts += ' -DVISIT_DATA_MANUAL_EXAMPLES=ON '
+configopts += ' -DVISIT_INSTALL_THIRD_PARTY=ON '
+configopts += ' -DVISIT_MPI_LIBS="mpich;pmi" '
+configopts += ' -DMPI_CXX_COMPILER=`which CC` '
+configopts += ' -DMPI_C_COMPILER=`which cc` '
+configopts += ' -DVISIT_PARALLEL_CXXFLAGS="-DPARALLEL -DMPICH_IGNORE_CXX_SEEK -I${CRAY_MPICH_DIR}/include"'
+configopts += ' -DVISIT_PARALLEL_LINKER_FLAGS="-L${CRAY_MPICH_DIR}/lib '
+configopts += ' ${CRAY_PMI_POST_LINK_OPTS}"; '
+
+prebuildopts  = ' cd build; '
+buildopts = 'install'
+
+dependencies = [ ('Python', '2.7.12'),
+                 ('Boost', '1.61.0', '-Python-2.7.12'),
+                 ('libxml2', '2.9.3', '-Python-2.7.12') ]
+
+# CMake and libxml2 dependencies with dummy toolchain
```

Comment:
Here you mention that you use the dummy toolchain, but you're not actually using it. 

FYI the magic syntax for using dummy is:
```
('libxml2', '2.9.3', '-Python-2.7.12', 'True')
```

My suggestion is to leave as it is now to avoid re-testing everything and just fix the comment.